src/journal/JournalTrimmer.cc

```diff
@@ -139,7 +139,8 @@ void JournalTrimmer::remove_set(uint64_t object_set) {
     librados::AioCompletion *comp =
       librados::Rados::aio_create_completion(ctx, NULL,
                                              utils::rados_ctx_callback);
-    int r = m_ioctx.aio_remove(oid, comp);
+    int r = m_ioctx.aio_remove(oid, comp,
+       (CEPH_OSD_FLAG_FULL_FORCE | CEPH_OSD_FLAG_FULL_TRY));
```

Comment:
Nit: either bring the ```oid, comp``` params down to the next line (and indent) or align the flags w/ the params on the previous line